packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -99,6 +99,28 @@ const cli = {
           return undefined;
         },
       },
+      seleniumServiceUrl: {
+        type: 'string',
+        describe: 'The address for the selenium service.',
+        default: () => {
+          if (process.env.SELENIUM_SERVICE_URL) {
+            return process.env.SELENIUM_SERVICE_URL;
+          }
+
+          return undefined;
+        },
+      },
+      port: {
+        type: 'number',
+        describe: 'The port mapping for the selenium service or the external selenium grid.',
+        default: () => {
+          if (process.env.SELENIUM_SERVICE_PORT) {
+            return process.env.SELENIUM_SERVICE_PORT;
```

Comment:
@mjhenkes OK with how I've named this env variable?

Reply:
I think we could get away without even offering the env's. We kept them for other variables due to precident in jenkins, but for new params we haven't been adding them.

Reply:
@mjhenkes I prefer to have them because it gives me a way to specify these things in my shell startup script so I don't have to type that parameter every time I run wdio.

Reply:
you could alternatively set the parameters on a 'localWindowsTest' npm script

Reply:
Yeah. I think it would be better to keep user/system-specific IP addresses out of package.json.

Reply:
I think it's fair to allow the minority of users who require this configuration to have as seamless an experience as mac users etc who run docker on localhost.